airflow/www/views.py

```diff
@@ -297,7 +297,7 @@ def dag_to_grid(dag: DagModel, dag_runs: Sequence[DagRun], session: Session):
     grouped_tis = {task_id: list(tis) for task_id, tis in itertools.groupby(query, key=lambda ti: ti.task_id)}
 
     def task_group_to_grid(item, grouped_tis, *, is_parent_mapped: bool):
-        if isinstance(item, AbstractOperator):
+        if not isinstance(item, TaskGroup):
```

Comment:
`trigger_rule` doesn't exist on `AbstractOperator`. Open to ideas on a better way to check if this is a task or a task group.